frontend/src/store/modules/visitors/actions.js

```diff
@@ -57,7 +91,7 @@ export default {
         }
         context.commit(SET_TABLE_FETCHING);
 
-        getTimeByPeriod(context.state.selectedPeriod)
+        return getTimeByPeriod(context.state.selectedPeriod)
```

Comment:
again, `getTimeByPeriod` doesn't return Promise, it returns object. So it won't have `then` method, so this action won't work